fieldservice_equipment_stock/views/fsm_order_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <!-- Fields Service Orders Form View -->
+    <record id="fsm_order_maintenance_form" model="ir.ui.view">
+        <field name="model">fsm.order</field>
+        <field name="inherit_id" ref="fieldservice.fsm_order_form" />
+        <field name="arch" type="xml">
+            <field name="equipment_id" position="attributes">
+                <attribute
+                    name="invisible"
+                >internal_type not in ['repair', 'maintenance', 'return']</attribute>
```

Comment:
Maybe I'm missing something, but where is the logic to set the internal_type to the fsm_order?